source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,50 @@
 		}
 	}
 }
+
+.is-style-features {
+	--wporg--style--feature--offset: 0.2em;
+	padding: 0;
+	list-style: none;
+	font-size: var(--wp--preset--font-size--heading-1);
+	font-weight: 200;
+	line-height: 1.1;
+
+	li {
+		overflow: hidden;
+		box-sizing: border-box;
+		height: 1.1em;
+		padding-top: var(--wporg--style--feature--offset);
+		padding-left: var(--wp--preset--spacing--50);
+		padding-right: var(--wp--preset--spacing--50);
+		color: #6682ff; // Hardcoded color for better contrast.
+		border-bottom: 1px solid var(--wp--preset--color--blueberry-3);
+		transition: all 0.2s ease-in-out;
+
+		a {
```

Comment:
`li` and `a` tend to be quite common elements in pages. Worth using direct descendant selectors for a minor performance gain (CSS is parsed from right to left)?

Reply:
This would leave the styles more heavily coupled to the markup structure though. We seem to have to attach a lot of styles to tags rather than classes, which are more robust and performant.

Reply:
I think we can be confident that the list block markup won't change, since it's a simple `ol,ul > li`, so I see no harm in tying to the structure. Also, there are no classes applied to the list items, so the only way to select them is through the tags. I can add a descendant selector to the `li`, though. The `a` i think I'll leave as-is, in case someone's bolded and linked the text or something :)

Reply:
Done in https://github.com/WordPress/wporg-parent-2021/pull/28/commits/27b7678d73ed3bd7e57199e4c8c78dc8258aaa80